testcases/kernel/syscalls/writev/writev06.c

```diff
@@ -138,9 +140,13 @@ int main(int argc, char **argv)
 			fail = 1;
 		}
 		if (fail) {
-			tst_resm(TINFO, "block 1 FAILED");
+			// This is a passing scenario, Hence, changing
```

Comment:
Shouldn't this say that this is a failing scenario?

Reply:
done